app/controllers/vacancies_controller.rb

```diff
@@ -46,7 +46,7 @@ def show
 
     @vacancy = VacancyPresenter.new(vacancy)
 
-    VacancyPageView.new(vacancy).track unless authenticated? || smoke_test?
+    VacancyPageView.new(vacancy).track unless authenticated? || smoke_test? || browser.bot?
```

Comment:
could you extract this check to some method like `not_tracked_case?` or something like that - might look cleaner if the list of cases we shouldn't track were to grow for some reason. I'll leave the method naming to you as my one was a poor example

Reply:
Thanks!

Reply:
For clarity - `authenticated? || smoke_test? || browser.bot` is the bit I would like to see extracted